opm/simulators/wells/BlackoilWellModel_impl.hpp

```diff
@@ -43,6 +48,7 @@
 #include <algorithm>
 #include <iomanip>
 #include <utility>
+#include <iostream>
```

Comment:
We should not use std::cout in the final version of the code. You can use OpmLog::info or OpmLog::debug, or you can 
use 
deferred_logger.info() or deferred_logger.debug().

Reply:
Removed all